LICENSE

```diff
@@ -13,17 +12,19 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of the copyright holder nor the names of its
-  contributors may be used to endorse or promote products derived from
-  this software without specific prior written permission.
+* Neither the name of 'pypi-mobans' nor the names of the contributors
+  may not be used to endorse or promote products derived from this software
```

Comment:
this `not` is wrong!

Reply:
I think `not` should not be there. double negative is positive.